model/variableSheet.h

```diff
@@ -21,21 +21,15 @@
 #ifndef VARIABLESHEET_H
 #define VARIABLESHEET_H
 #include <cairoSurfaceImage.h>
-#include <item.h>
+#include <parVarSheet.h>
 
 namespace minsky
 {
 	 
-  class VariableSheet: public ecolab::CairoSurface, public ItemT<VariableSheet>
+  class VariableSheet: public ecolab::CairoSurface, public ParVarSheet
```

Comment:
I'm not sure I understand these comments.

ParVarSheet is the base class for derived classes ParameterSheet and VariableSheet.

Since I have introduced rank 3 or higher parameters into ParVarSheet::draw(), I don't think that I can use the code in Sheet here, if that is what you meant?

Reply:
Oh, sorry, you've already resolved this conversation. Still getting used to Github...